docs/concepts/personal-access-token.mdx

```diff
@@ -1,31 +0,0 @@
----
-id: personal-access-token
-title: Personal Access Tokens
----
-
-You need to access the Ory administrative APIs if you want to:
-
-- [list](../cli/ory-identities-list.md),
-- [update](../cli/ory-identities-get.md) or
-- [delete](../cli/ory-identities-delete.md) identities,
-- use the [Ory CLI](../guides/cli/10_config-with-cli.mdx) to manage your project
-
-To access administrative APIs at Ory you need to authenticate as an administrator. Ory offers Personal Access Tokens to make this
-process easier for development. You can use PATs to access the admin API from the command line or via scripts.
-
-:::note
-
-Ory's Personal Access Tokens are always prefixed with `ory_pat_`, making them easier to identify in code analysis.
-
-:::
-
-Personal Access Tokens are bound to a project. They don't work for other projects than the one they have been created for.
```

Comment:
These two pieces are important, please re-add this. It is very common to have a docs page about this. In fact, I don't know a single documentation that doesn't talk about this. Example: https://stripe.com/docs/keys

Reply:
Brought the document back and updated it:
`docs/concepts/personal-access-token.mdx`